### ISSUE TITLE
the record shortcut button is not accessible to voiceover users under big sir

### ISSUE THREAD

**Issue by marrie** [gbammc/Thor#41]
# summary

When voiceover is on, I cannot access the record shortcut button in thor.. Voiceover can be accessed with command plus f5.

# steps

1. Launch voiceover with command f5.
2. Launch thor
3. Go to the table and add an app.
4. 4. Click, or try and click "record shortcut.
I notice it is not there when in the table. I'm using the following for my operating system.

ProductName:	macOS
ProductVersion:	11.4
BuildVersion:	20F71

<img width="1440" alt="a screenshot of the main table in  thor showing the apps list I have" src="https://user-images.githubusercontent.com/3614487/123466664-73422680-d5a4-11eb-91ca-26f71f90a7a8.png">e

Thanks.

**Comment by gbammc** [comment]
I can add a new shortcut when VoiceOver is on. Do you want to add a shortcut for the Backblaze app? Did it install in the Application Folder?

**Comment by marrie** [comment]
Hello. That is worth a try. If a screen reader is detected, then make 
the record shortcut button accessible. For now I have workflows doing 
what this app does, but this app is so much easier. Lol! No the shortcut 
was never made due tot he missing record shortcut button.

Thanks.
